lightspeed/modules/proc_login-vscode-extension.adoc

```diff
@@ -13,18 +13,14 @@ After installing and configuring the VS Code extension, you can log in to the An
 +
 Users are authenticated using your {PlatformName} login. 
 +
-To sign in, you can use the *Connect* button or the *Accounts* button, and use the option *Sign in with Ansible Lightspeed to use Ansible* from VS Code. Once prompted, select *Log in with {PlatformNameShort}*, and log in with the authorization mechanism that your automation controller is configured with. 
+To sign in, you can use the *Connect* button in the Ansible Lightspeed view, or the *Sign in with Ansible Lightspeed to use Ansible* option in the *Accounts* menu. Once prompted in the browser, select *Log in with Ansible Automation Platform*, and log in with the authorization mechanism that your automation controller is configured with. 
 
 * *Ansible Lightspeed cloud service*
 +
 Users are authenticated using {RHSSO} ({RHSSOshort}).
 +
-To sign in, you can use the *Connect* button or the *Accounts* button and then select one of the following options:
+To sign in from VS Code, you can use the *Connect* button in the Ansible Lightspeed view, or the *Sign in with Ansible Lightspeed to use Ansible* option in the *Accounts* menu. Follow the on-screen prompts to log in and access the Ansible Lightspeed service using your ({RHSSOshort}).
```

Comment:
I don't know that you need the parenthesis in this instance ({RHSSOshort}) since you're using the acronym and not the full name followed by the acronym.

Reply:
Good catch, updated.